loom/engine/bindings/loom/lmHTTPRequest.cpp

```diff
@@ -75,7 +75,7 @@ class HTTPRequest {
         // Warn if this object is garbage collected while a requet is pending
         if (requestPending)
         {
-            lmLog(gHTTPRequestLogGroup, "WARNING: HTTPRequest object with url \"%s\" garbage collected before completing a pending request. Ensure references to HTTPRequest objects are maintained while requests are pending.", url.c_str())
+            lmLogWarn(gHTTPRequestLogGroup, "HTTPRequest object with url \"%s\" garbage collected before completing a pending request. Ensure references to HTTPRequest objects are maintained while requests are pending.", url.c_str())
```

Comment:
Yes. Everything >= Info is visible by default. Of course you can explicitly filter to only see errors, in which case this won't show, but that's your fault.